tests/taskTests/codeDeployDeployApplication/codeDeployDeployApplication-test.ts

```diff
@@ -147,7 +161,7 @@ describe('CodeDeploy Deploy Application', () => {
                     }
                 }
 
-                return emptyPromise
+                return sleepOneSecPromise
```

Comment:
The PR description has great context, which is lost on a future maintainer looking at this line of code.
Can you add a brief one-line comment explaining the significance of sleepOneSecPromise wherever you use it in this file?

Reply:
> The PR description has great context, which is lost on a future maintainer looking at this line of code.

Also helps to include the PR description in the commit message. That allows `git log` (and related tools) to find context from the line of code.